README.md

```diff
@@ -9,14 +9,41 @@ Before you can use the Miking system, you need to install
 [OCaml](https://ocaml.org/) and the
 [OPAM](https://opam.ocaml.org/) package manager.
 
-
-After the installation, you need to install the `opam` packages `dune`, `batteries`, and `linenoise` by running the following:
+After the installation, you need to install the OCaml multicore compiler by
+following these steps:
+* Clone the `miking-lang/multicore-ocaml` repository by running the following in
+  your choice of directory:
+  ```
+  git clone https://github.com/miking-lang/ocaml-multicore.git
+  ```
+* Make sure you are on the `no-effect-syntax` branch in the
+  `miking-lang/multicore-ocaml` repository:
+  ```
+  git checkout no-effect-syntax
+  ```
```

Comment:
If the commands are copy-pasted, then this command will fail because one has to change directory before running this command. Perhaps you could also add a line `cd ocaml-multicore` to this command?

Reply:
Thanks, prefixed a `cd` command